src/components/DataDetails.vue

```diff
@@ -148,43 +160,48 @@ export const DefaultActionKeys = [
     name: 'View Item',
     iconKey: 'view',
     color: 'primary',
-    handler() {
-      const { value: items } = this;
-      download(this.girderRest.apiRoot, items[0]._modelType, items[0]._id, '?contentDisposition=inline');
+    url(apiRoot, items) {
+      return generateUrl(apiRoot, items[0]._modelType, items[0]._id, '?contentDisposition=inline');
+    },
+    handler(girderRest, items) {
```

Comment:
It seems like we can delete these `handler`s now, or am I missing something? Like we should make the `handler` optional, and change the docstring on L149 to reflect the new behavior. Then we could remove the `@click.prevent`, which seems like a good idea. At best, we want to `.stop`, not `.prevent` on something with an href.

Reply:
`handler` still has a distinct implementation in the `Delete` action, so I left it in place. An alternative would be to change `handleAction` to test if there is a `url`, and if so, do `download(action.url(girderRest.apiRoot, items));` directly there. Then we could delete the stupid-looking `hander`s that I left hanging around. I didn't do that because I thought the link between having a `url` and being downloadable wouldn't be obvious.

I missed the docstring, I will update it to reflect the new schema.

I'm pretty sure that as long as we still have `handler`, the event modifier should still be `.prevent`. `.stop` will stop the event from bubbling up to the girder-detail-list.

Reply:
I'm just suggesting that we make `handler` optional, such that these actions would either implement a `handler` or a `url`, but not both. Then inside `handleAction`, we only call the `handler` if one is set. Does that make sense? If we do that, we don't need `.stop` or `.prevent` at all.

> I'm pretty sure that as long as we still have handler, the event modifier should still be .prevent. .stop will stop the event from bubbling up to the girder-detail-list.

In that case, stopping the event bubbling up is exactly what we want, since it's the ancestor element that actually handles the event. But again, I think the right solution is to use neither and let the event process as normal.